fvm/src/gas/price_list.rs

```diff
@@ -860,6 +860,15 @@ impl PriceList {
         self.wasm_rules.memory_expansion_per_byte_cost * grow_memory_bytes as i64
     }
 
+    /// Returns the gas required for initializing tables.
+    pub fn init_table_gas(&self, min_table_elements: u32) -> Gas {
+        // Each element reserves a `usize` in the table, so we charge 8 bytes per pointer.
+        // https://docs.rs/wasmtime/2.0.2/wasmtime/struct.InstanceLimits.html#structfield.table_elements
+        self.wasm_rules.memory_expansion_per_byte_cost
+            * (min_table_elements as i32)
+            * (core::mem::size_of::<*const ()>() as i32)
```

Comment:
We need this charge to be architecture independent, so let's just charge for 8 bytes (64 bits).

Reply:
Thanks. I had the same thought but I wasn't sure if it's possible to run this with 32 bit systems.

Reply:
Well... Ok, it's not. But I'd still prefer to just hard-code this.